examples/with-aws-lambda/backend/config.mjs

```diff
@@ -0,0 +1,70 @@
+import ThirdPartyEmailPassword from "supertokens-node/recipe/thirdpartyemailpassword";
```

Comment:
we should remove everything from this folder and just have a readme that links to the node repo example since that's tested and all + don't want to maintain the same thing here and there